examples/typescript-interop/node/platform/main.roc

```diff
@@ -0,0 +1,10 @@
+platform "typescript-interop"
+    requires {} { main : Str -> Str }
+    exposes []
+    packages {}
+    imports [Json]
```

Comment:
nit: no need to still import json

Reply:
Fixed in 8b65402e6c72425c23830c4e3fb21cc151531d4f